terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf

```diff
@@ -689,6 +689,7 @@ module "salt-migration-minion" {
   auto_connect_to_master  = true
   use_os_released_updates = false
   ssh_key_path            = "./salt/controller/id_rsa.pub"
+  install_salt_bundle = false
```

Comment:
have you tried this locally? from what I remember, we added the salt bundle because there were issues when we didn't have it

Reply:
I deployed the minion without salt-bundle in the 4.3.12 BV env, using terraform from the dedicated Jenkins workspace. It correctly contacts the server and its Salt key appears ready to be accepted.
Once that's done, it is onboarded and seems to have been working fine so far.

Maybe those are Uyuni specific issues ?

Reply:
FMPOV you can leave it in, Paolo, and test it in the Uyuni BV once this PR gets merged.

Reply:
same fears as Vanda :)

Reply:
Ok, this can stay parked here for a while and I can try this out in Uyuni BV to see what happens.
Tests will just do a venv-salt-minion installation anyway but nothing should change.

Reply:
`install_salt_bundle` option only install venv-salt-minion instead of salt-minion during the cloud init. This salt will then be used for the salt states and normally clear after the deployment.